### DISCUSSION TITLE
Only one instance - how?

### DISCUSSION THREAD

**Discussion by cipeczka** [probonopd/MiniDexed#343] [Q&A] [answered]
How configure dx to play only one instance?
Thx

**Comment by craigyjp** [comment]
Set the number of TGs to 1

**Reply by cipeczka** [reply]
Where set it? 
Thanks!

**Reply by probonopd** [reply]
Set the MIDI channel of TG 2 to 4 to a channel other than 1 and other than OMNI. This can be done either in the `performance.ini` or using the LCD screen and menu on the MiniDexed device itself.

**Comment by diyelectromusic** [comment, marked as answer]
It is all in the performance.ini file.  The default config plays quite a few tone generators at the same time.  The way to disable each TG is to set the respective MidiChannel settings to 0, which turns that TG off for MIDI, or set them all to different MIDI channels.  At present they are all set to 255 which is MIDI Omni mode (i.e. respond to all MIDI channels).

Note that the system will automatically only configure a single TG if running on a non-multicore Raspberry Pi (e.g. RPi V1).

I must admit that the default performance.ini configuration is a little "rich" for those first trying the system.  I think we probably need a simpler one to get people started - e.g. all starting on the same bank/voice, no effects, no detune, no panning, etc and with each TG mapped to MIDI channels 1-8 - that kind of thing...
